Library/Homebrew/test/support/fixtures/cask/Casks/adobe-air.rb

```diff
@@ -0,0 +1,10 @@
+cask 'adobe-air' do
+  name 'Adobe Air'
```

Comment:
The name should be in between `sha256` and `url`. Might be worth running `brew cask style` on these.

Reply:
`brew cask style` and `brew style` didn't pick-up any offenses for these test casks. Should I look at including them in the style check, in a follow-up PR?

For now, I made the corrections based on https://github.com/Homebrew/homebrew-cask/tree/master/doc/cask_language_reference#stanza-order

Reply:
Yep, that would be great.